node/tests/utils.rs

```diff
@@ -163,7 +165,11 @@ impl MASQNode {
 
     #[allow(dead_code)]
     pub fn wait_for_log(&mut self, pattern: &str, limit_ms: Option<u64>) {
-        Self::wait_for_match_at_directory(pattern, self.data_dir.as_path(), limit_ms);
```

Comment:
Maybe change `pattern` for `regex` or `regex_pattern`.

Reply:
renamed